CorsixTH/Lua/objects/reception_desk.lua

```diff
@@ -92,48 +92,74 @@ function ReceptionDesk:onClick(ui, button)
 end
 
 function ReceptionDesk:tick()
+  --! Handle a patient at the reception desk.
+  --!param patient The patient at the front of queue.
+  --!param new (bool) Have they just entered the hospital?
+  local function handlePatient(patient, new)
```

Comment:
```suggestion
  local function handlePatient(patient, is_new)
```
By prepending "is" or "has" it becomes more obvious the variable holds a boolean value.